src/test/java/eu/deltasource/model/OwnerTest.java

```diff
@@ -0,0 +1,37 @@
+package eu.deltasource.model;
+
+import eu.deltasource.exception.InvalidInputException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class OwnerTest {
+
+    @Test
+    void testIfABlankFirstNameIsEntered() {
+        //GIVEN
+        Assertions.assertThrows(InvalidInputException.class, () -> {
+            Owner owner = new Owner("          ", "Ivanov");
+        });
+    }
+
+    @Test
+    void testIfABlankLastNameIsEntered() {
+        //GIVEN
+        Assertions.assertThrows(InvalidInputException.class, () -> {
+            Owner owner = new Owner("Ivan", "  ");
+        });
+    }
+
+    @Test
+    void testIfABlankIdIsEntered() {
+        //GIVEN
+        BankInstitution bankInstitution = new BankInstitution("dsk", "ivan 2");
+
+        Assertions.assertThrows(InvalidInputException.class, () -> {
+            Owner owner = new Owner("Ivan", "Ivanov");
+            BankAccount bankAccount = new BankAccount(owner, "     ", bankInstitution, "odsjdspf", "GBP", -30, "CurrentAccount");
```

Comment:
According to the test you are testing for blank id case scenario, but you are passing from what it seems several invalid parameters to the BankAccount constructor.  You should pass everything correct, except what you test for your case.